data/label_studio.py

```diff
@@ -37,3 +42,46 @@ def delete_tasks(self, id_list: list) -> None:
                 headers={"Authorization": f"Token {self.client.api_key}"}
             )
         print(f"{len(id_list)}개의 task 삭제 완료!")
+
+    def get_labelled_tasks(self) -> list:
+        labelled_tasks = []
+        for task in self.project.get_labeled_tasks():
+            scoring_criterion_dict = {
+                criterion['alias']: criterion['value']
+                for criterion in task['data']['sim_list']
+            }
+            keyword_criterion_dict = {
+                criterion['alias']: criterion['value']
+                for criterion in task['data']['keyword_list']
+            }
+            annotations: dict = task['annotations'][-1]['result']
+
+            # 선택된 후보가 여러개인 경우, 한개인 경우, 없는 경우 세가지로 나뉨
+            scoring_annotation, keyword_annotation = [], []
+
+            for annotation in annotations:
+                if isinstance(annotation['value'], dict):  # 선택지가 여러개인 경우
+                    for choice in annotation['value']['choices']:
+                        choice = choice[0]
```

Comment:
```suggestion
                    for choice, *res in annotation['value']['choices']:
```

for 문에서 사용되는 변수에 재할당하는 것은 좋아보이지 않아요.

Reply:
와 이거 모죠 굉장히 파이써닉 하네요!! 너무 좋슴다

Reply:
@ekzm8523 생각해보니까 python에서 사용하지 않는 변수들은 보통 _ ( underscore ) 로 사용하는걸로 기억하는데

```python
for choice, *_ in annotation['value']['choices']:
    pass 
```

위 처럼 해도 될 것 같아요

Reply:
뭔가 *res 처럼 쓰면, res가 의미있게 사용될 것 같은 느낌이랄까요

Reply:
맞습니다~~ 조아요~